app/src/main/java/com/vero/tasky/agenda/domain/usecase/GetFullAgendaUseCase.kt

```diff
@@ -0,0 +1,30 @@
+package com.vero.tasky.agenda.domain.usecase
+
+import androidx.work.*
+import com.vero.tasky.agenda.domain.worker.GetFullAgendaWorker
+import javax.inject.Inject
+
+class GetFullAgendaUseCase @Inject constructor(
+    private val workManager: WorkManager
```

Comment:
UseCase / Domain layer shouldn't know anything about Android stuff

You could create an interface, and then the implementation on Data where you create the WorkManager
Kinda like what you did with the EmailMatcher